spec/services/shopify_import/invoker_spec.rb

```diff
@@ -2,7 +2,11 @@
 
 RSpec.describe ShopifyImport::Invoker do
   describe '#import!' do
-    ROOT_IMPORTERS = [ShopifyImport::Importers::ProductsImporter, ShopifyImport::Importers::CustomersImporter].freeze
+    ROOT_IMPORTERS = [
```

Comment:
Do you think we should move this to the Invoker? we could use it both there in `#import!` and here.